### PR TITLE
Use LOG_TARGET in consensus related crates

### PR DESCRIPTION
**Pull request by davxy**
Partially address https://github.com/paritytech/substrate/issues/12873

---

This PR defines the log target for Substrate consensus algorithms in the root file of the respective crates.
(e.g. for `sc-consensus-babe` it is defined in `client/cosensus/babe/lib.rs`)

---

### Release Note worth mentioning

The log target for `grandpa` consensus has been changed from the legacy name `"afg"` to the current name of the algorithm, i.e. `"grandpa"`

### CODE REVIEW

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-check-benches
**Logs**: https://gitlab.parity.io/parity/mirrors/substrate/-/jobs/2136187

File: client/consensus/common/src/import_queue.rs [hunk 1]
```diff
@@ -45,6 +45,8 @@ use crate::{
 pub use basic_queue::BasicQueue;
 use sp_consensus::{error::Error as ConsensusError, BlockOrigin, CacheKeyId};
 
+const LOG_TARGET: &str = "import-queue";
```
Comment (davxy) [inline client/consensus/common/src/import_queue.rs:48]: The logs target here were "sync", but maybe "import-queue" is more appropriate?

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-check-benches
**Logs**: https://gitlab.parity.io/parity/mirrors/substrate/-/jobs/2136279

File: client/finality-grandpa/src/lib.rs [hunk 2]
```diff
@@ -93,10 +93,12 @@ use std::{
 	time::Duration,
 };
 
+const LOG_TARGET: &str = "grandpa";
```
Comment (davxy) [inline client/finality-grandpa/src/lib.rs:96]: This once was "afg". 
My understanding is that AFG was the original GRANDPA name, but maybe is more appropriate to use the actual name

File: frame/grandpa/src/lib.rs [hunk 3]
```diff
@@ -70,6 +70,8 @@ pub use equivocation::{
 
 pub use pallet::*;
 
+const LOG_TARGET: &str = "runtime::grandpa";
```
Comment (davxy) [inline frame/grandpa/src/lib.rs:73]: Again, this was "runtime::afg"

File: client/consensus/common/src/import_queue.rs (hunk 1 above)
Comment (bkchr) [inline client/consensus/common/src/import_queue.rs:48]:
```suggestion
const LOG_TARGET: &str = "sync::import-queue";
```

The good with `sync` is that putting `sync=trace` also gives you these things into the log.

File: client/finality-grandpa/src/lib.rs (hunk 2 above)
Comment (bkchr) [reply client/finality-grandpa/src/lib.rs:96]: I think it is fine. Maybe @andresilva has a different opinion.

**Review by bkchr — APPROVED** [review]

**Pull request merged**